### PR TITLE
Various RTL fixes

### PR DESCRIPTION
**Pull request by yosilevy**
## Breaking change

Gauge functional fix - wasn't updating when label was changed and not value (happens when editing in editor)

Various RTL fixes:
ha-chip fix - discovered in alarm card
textfield additional cases
alarm panel - keypad always LTR now
light card - more info moved to right location
user picker - switch to our list item which supports RTL
media control - fix alignments

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-textfield.ts [hunk 1]
```diff
@@ -54,11 +54,19 @@ export class HaTextField extends TextFieldBase {
...
       .mdc-text-field__affix--suffix {
         padding-left: var(--text-field-suffix-padding-left, 12px);
         padding-right: var(--text-field-suffix-padding-right, 0px);
-        padding-inline-start: var(--text-field-suffix-padding-left, 12px);
-        padding-inline-end: var(--text-field-suffix-padding-right, 0px);
+      }
+      .mdc-text-field__affix--suffix,
+      .mdc-text-field--with-leading-icon {
+        padding-inline-start: var(--text-field-suffix-padding-left, 0px);
+        padding-inline-end: var(--text-field-suffix-padding-right, 12px);
```
Comment (bramkragten) [inline src/components/ha-textfield.ts:69]: 0 and 12 are now flipped?

File: src/components/ha-chip.ts [hunk 2]
```diff
@@ -67,8 +67,7 @@ export class HaChip extends LitElement {
         color: var(--ha-chip-icon-color, var(--ha-chip-text-color));
       }
       .mdc-chip.mdc-chip--selected .mdc-chip__checkmark,
-      .mdc-chip.no-text
-        .mdc-chip__icon--leading:not(.mdc-chip__icon--leading-hidden) {
+      .mdc-chip .mdc-chip__icon--leading:not(.mdc-chip__icon--leading-hidden) {
```
Comment (bramkragten) [inline src/components/ha-chip.ts:70]: What was the reason for this change?

File: src/components/ha-textfield.ts (hunk 1 above)
Comment (yosilevy) [reply src/components/ha-textfield.ts:69]: Sorry. Made a mess there. Separated, fixed, verified.

File: src/panels/logbook/ha-logbook-renderer.ts [hunk 3]
```diff
@@ -86,18 +83,13 @@ class HaLogbookRenderer extends LitElement {
...
-    }
-  }
-
   protected render(): TemplateResult {
     if (!this.entries?.length) {
       return html`
-        <div class="container no-entries" .dir=${emitRTLDirection(this._rtl)}>
+        <div
+          class="container no-entries"
+          .dir=${computeRTLDirection(this.hass)}
```
[the remark this answers is not in the document]
Comment (yosilevy) [reply src/panels/logbook/ha-logbook-renderer.ts:91]: Removed

File: src/components/user/ha-user-picker.ts [hunk 4]
```diff
@@ -48,14 +49,14 @@ class HaUserPicker extends LitElement {
           : ""}
         ${this._sortedUsers(this.users).map(
           (user) => html`
-            <mwc-list-item graphic="avatar" .value=${user.id}>
+            <ha-clickable-list-item graphic="avatar" .value=${user.id}>
```
[the remark this answers is not in the document]
Comment (yosilevy) [reply src/components/user/ha-user-picker.ts:52]: Created a new HaListItem and refactored clickable to derive from it.

File: src/components/ha-chip.ts (hunk 2 above)
Comment (yosilevy) [reply src/components/ha-chip.ts:70]: It is needed also for the regular case not just no-text.

**Comment by yosilevy** [comment]
@bramkragten ready for you

**Review by bramkragten — APPROVED** [review]

**Pull request merged**